common/pom.xml

```diff
@@ -114,5 +114,10 @@
             <version>4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.fluentd</groupId>
```

Comment:
Do we really need to pull this into all modules? Can this be moved to serving only?

Reply:
AuditLogger is used in both Core and Serving, and it exists in common module, so I think that's not possible.